crates/llms/src/anthropic/types_stream.rs

```diff
@@ -123,7 +126,7 @@ impl Delta {
     pub fn into_completion(
         self,
         role: &Option<MessageRole>,
-        tool_content: Option<(i32, ContentBlockToolUse)>,
```

Comment:
Bug meant that many tool chunks weren't all being aggregated. Anthropic only ever has index=0 (can't do multiple streams in parallel).